### PR TITLE
doc() : Misc documentation updates

### PR DESCRIPTION
**Pull request by jdescottes-talend**
**What is the problem this PR is trying to solve?**
Some documentations are not up-to-date or have some flaws.

**What is the chosen solution to this problem?**
Add some documentation changes

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/cmf/src/bootstrap.md [hunk 1]
```diff
@@ -37,6 +38,9 @@ cmf.bootstrap({
 | middlewares | Array | undefined | Redux middlewares |
 | storeCallback | function | undefined | Let you call a function once the store is created |
 | AppLoader | React Component | undefined | Let you define the React component to use to show the app is currently loading (waiting for the settings) |
+| saga | generator function | undefined | An independant top level saga of the app |
+| sagaRouterConfig | Object | undefined | (DEPRECATED) A route dictionnary of path (as key) associatied with saga (as value)
```
Comment (jmfrancois) [inline packages/cmf/src/bootstrap.md]: missing last |

File: packages/cmf/src/bootstrap.md [hunk 2]
```diff
@@ -45,18 +49,19 @@ You apps may be splitted into different modules, like a core set of components a
...
-import saga from './saga';
 import sagas from './sagas';
 
-cmf.boostrap({
+cmf.bootstrap({
 	components,
-	saga,
-	sagas,
-	modules: [containersModule],
+	saga: sagas.appSaga,
```
Comment (jmfrancois) [inline packages/cmf/src/bootstrap.md]: ?? I don t understand this. 
The main saga should never leave in sagas. You will not do DI with it

File: packages/cmf/src/bootstrap.md [hunk 3]
```diff
@@ -45,18 +49,19 @@ You apps may be splitted into different modules, like a core set of components a
...
+	sagaRouterConfig: sagas.routes,
```
Comment (jmfrancois) [inline packages/cmf/src/bootstrap.md]: same point here

File: packages/cmf/src/bootstrap.md (hunk 1 above)
Comment (jdescottes-talend) [reply packages/cmf/src/bootstrap.md]: This has been updated

File: packages/cmf/src/bootstrap.md (hunk 2 above)
Comment (jdescottes-talend) [reply packages/cmf/src/bootstrap.md]: Can you update the 2 examples with what you think is appropriate about folder structure/export style (default/named, things grouped or not) about the sagas ?

**Pull request closed without merging**